genomedata/_load_data.py

```diff
@@ -13,6 +13,7 @@
 from subprocess import PIPE, Popen
 
 from . import __version__
+from ._load_data_c_ext import load_data_from_stdin
```

Comment:
File docstring needs fixing.

Can we change the name of this file to `load_data.py` and then the extension can be `_load_data` as is typical for C extensions?

Reply:
What about all the other files such as `_load_seq.py`? The entire python genomedata library effectively starts with `_` to presumably have users to solely use the main `__init__.py`/Genome interface.

Reply:
How about `_c_load_data`? There is some precedent for using c at the beginning for an extension module (for example `cProfile`).

Reply:
Sure I will update the name accordingly